ui/src/pages/Alert/Channel/ChannelDrawer.tsx

```diff
@@ -0,0 +1,164 @@
+import { alert } from '@/api';
+import AlertDrawer from '@/components/AlertDrawer';
+import { CHANNEL_TYPE_OPTIONS } from '@/constants';
+import { QuestionCircleOutlined } from '@ant-design/icons';
+import { useRequest } from 'ahooks';
+import type { DrawerProps } from 'antd';
+import { Button, Form, Input, Select, Space } from 'antd';
+import { useEffect } from 'react';
+
+/**
+ * ChannelDrawer has three states: creation,editing and pure display
+ * Editing state and pure display state can be switched between each other
+ */
+interface ChannelDrawerProps extends DrawerProps {
+  status: Alert.DrawerStatus;
+  name?: string;
+  setStatus?: (status: Alert.DrawerStatus) => void;
+}
+
+const { TextArea } = Input;
+
+export default function ChannelDrawer({
+  status,
+  name,
+  onClose,
+  setStatus,
+  ...props
+}: ChannelDrawerProps) {
+  const [form] = Form.useForm();
+  const { run: getReceiver } = useRequest(alert.getReceiver, {
+    manual: true,
+    onSuccess: ({ successful, data }) => {
+      if (successful) {
+        form.setFieldsValue({
+          ...data,
+        });
+      }
+    },
+  });
+  const { data: listReceiversRes } = useRequest(alert.listReceivers, {
+    manual: true,
```

Comment:
Why set `manual: true` here while no `run` used in following part? If you don't run manually, the `data` (or `listReceiversRes`) will always be null.